angel_system/data/medical/load_bbn_data.py

```diff
@@ -207,42 +231,61 @@ def save_as_kwcoco(classes, data, save_fn="bbn-data.mscoco.json"):
     dset.dump(dset.fpath, newlines=True)
 
 
-def activity_label_fixes(activity_label, target):
-    if activity_label == "put_tourniquet_around":
-        label = "place-tourniquet"
-        label_id = 1
-    if activity_label == "pulls_tight":
-        label = "pull-tight"
-        label_id = 2
-    if activity_label == "secures" and target == "velcro_strap":
-        label = "apply-strap-to-strap-body"
-        label_id = 3
-    if activity_label == "twist" and target == "windlass":
-        label = "turn-windless"
-        label_id = 4
-    if (
-        activity_label == "locks_into_windlass_keeper"
-        or activity_label == "lock_into_windlass_keeper"
-    ):
-        label = "lock-windless"
-        label_id = 5
-    if (
-        activity_label == "wraps_remaining_strap_around"
-        or activity_label == "wrap_remaining_strap_around"
-    ):
-        label = "pull-remaining-strap"
-        label_id = 6
-    if activity_label == "secures" and target == "windlass":
-        label = "secure-strap"
-        label_id = 7
-    if activity_label == "writes_on" and target == "tourniquet_label":
-        label = "mark-time"
-        label_id = 8
-
+def activity_label_fixes(task, activity_label, target):
```

Comment:
This is hacky - the way BBN gave us activity GT, we parse it like this in angel_system/data/medical/load_bbn_data.py:
```
data = line.split("\t")

            # Find frame filenames
            start_frame = int(data[0])
            end_frame = int(data[1])

# Determine activity
            activity_str = data[2].strip().split(" ")
            hand = activity_str[0]
            activity = activity_str[1]
            target = activity_str[2] if len(activity_str) > 2 else None
```
Example output:
```
212     1128    hands      apply_pressure_to        casualty_wound
```
(Paraphrased)
"start frame"
"end frame"
"which hand is doing the activity" ("both" = "hands")
"activity" (generic, could apply to R18 or M2)
"target"

Reply:
Should we add some text like this to the docstring for this function?